Tests/js/models/assets/ez-locationmodel-tests.js

```diff
@@ -53,7 +92,32 @@ YUI.add('ez-locationmodel-tests', function (Y) {
         tearDown: function () {
             this.model.destroy();
             delete this.model;
-        }
+        },
+
+        "The content info should be instance of eZ.ContentInfo": function () {
+            var contentInfoStruct = this.loadResponse.Location.ContentInfo;
+
+            this.model.set('contentInfo', contentInfoStruct);
+            Y.Assert.isInstanceOf(
+                Y.eZ.ContentInfo,
+                this.model.get('contentInfo'),
+                'Content Info should be instance of eZ.ContentInfo'
+            );
+            Y.Assert.areEqual(
+                contentInfoStruct.Content.Name,
+                this.model.get('contentInfo').get('name'),
+                'Should instantiate content info with the name'
+            );
+        },
+
+        "The content info should be instance of eZ.ContentInfo when set to undefined": function () {
+            this.model.set('contentInfo', undefined);
+            Y.Assert.isInstanceOf(
+                Y.eZ.ContentInfo,
+                this.model.get('contentInfo'),
+                'ContentInfo should be instance of eZ.ContentInfo'
+            );
+        },
```

Comment:
and in the current code state, it will most likely fail because in the Location model you need to either change ATTRS_REST_MAP or to implement a custom _parseStruct so that the contentInfo is actually read